src/js/components/Meter/Meter.js

```diff
@@ -19,9 +20,12 @@ const deriveMax = (values) => {
 const Meter = forwardRef(
   (
     {
+      'aria-label': ariaLabel,
       background = { color: 'light-2', opacity: 'medium' },
       color,
       direction = 'horizontal',
+      max,
```

Comment:
`max` documentation states "The maximum value for the Meter." Does this imply that if I specify `max={10}`, then set `value={20}`, the 20 won't be respected? If not, should the definition of max be clarified?

Reply:
correct we can clarify our documentation. 

so if we have the following: 
```
      <Meter type="bar" max={50} value={60} />
```

The whole meter will be filled. 

<img width="454" alt="Screenshot 2024-11-25 at 7 18 01 PM" src="https://github.com/user-attachments/assets/cb565d76-ec9d-4a6c-a6dc-1a2949f61518">